MCprep_addon/spawner/meshswap.py

```diff
@@ -107,7 +108,7 @@ def move_assets_to_excluded_layer(context, collections):
 
 def update_meshswap_path(self, context):
 	"""for UI list path callback"""
-	conf.log("Updating meshswap path", vv_only=True)
+	env.log("Updating meshswap path", vv_only=True)
```

Comment:
I wonder if we should just create a .debug function instead, and nix the use of a vv_only flag.